packages/cactus-verifier-client/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@hyperledger/cactus-verifier-client",
```

Comment:
@outSH May I recommend using a more specific package name like `cactus-verifier-client-besu-socketio`?

Reply:
@petermetz This is a common interface for all ApiClients, not only besu one. For now it only supports besu and our "legacy-socketio" fabric, go-eth and sawtooth validators, but in near future we'll add support for quorum and possibly other validators, and we want to make them compatible with common verifier interface (introduced in cmd-socketio package).

Reply:
@outSH Understood, `cactus-verifier-client-legacy-socketio` then?

Reply:
@petermetz Once again, this is not an interface for `legacy-socketio` (*-socketio* connectors), but in the end to all the connectors, including openapi ones. This is the verifier component documented in the whitepaper, but modified to support all connectors / validators.